era5_monthly_averages_2008/Manifest.toml

```diff
@@ -45,7 +47,7 @@ version = "0.1.3"
 
 [[deps.ClimaArtifactsHelper]]
 deps = ["ArtifactUtils", "Downloads", "Pkg", "REPL", "SHA"]
-path = "/Users/treddy/.julia/dev/ClimaArtifacts/ClimaArtifactsHelper.jl"
+path = "/home/treddy/.julia/dev/ClimaArtifacts/ClimaArtifactsHelper.jl"
```

Comment:
Is there a way we can avoid hardcoding the path here?

Reply:
Good catch. Fixed it now